src/routes.json

```diff
@@ -153,14 +153,14 @@
     },
     {
         "name": "help",
-        "pattern": "^/help/?$",
-        "routeAlias": "/help",
+        "pattern": "^/help/?(\\?.*)?$",
+        "routeAlias": "^/help/?(\\?.*)?$",
```

Comment:
The `^` is not necessary in the `routeAlias`, because it is added by the configuration script